<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nightwood - Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <layout class="QHBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="NavBar" name="navBar" native="true">
     <property name="minimumSize">
      <size>
       <width>220</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="contentLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="leftMargin">
      <number>35</number>
     </property>
     <property name="topMargin">
      <number>35</number>
     </property>
     <property name="rightMargin">
      <number>35</number>
     </property>
     <property name="bottomMargin">
      <number>35</number>
     </property>
     <item>
      <spacer name="topSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="pageTitle">
       <property name="styleSheet">
        <string notr="true">font-size: 24px; font-weight: bold; color: #2d3436; margin-bottom: 10px;</string>
       </property>
       <property name="text">
        <string>Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="settingsContentLayout">
       <property name="spacing">
        <number>30</number>
       </property>
       <item>
        <widget class="QWidget" name="passphraseWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>400</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget#passphraseWidget {
  background-color: white;
  border: 1px solid #dfe6e9;
  border-radius: 12px;
  padding: 30px;
}</string>
         </property>
         <layout class="QVBoxLayout" name="passphraseLayout">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="passphraseSectionTitle">
            <property name="styleSheet">
             <string notr="true">font-size: 18px; font-weight: bold; color: #2d3436; margin-bottom: 10px;</string>
            </property>
            <property name="text">
             <string>Change Passphrase</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="currentPassphraseLabel">
            <property name="styleSheet">
             <string notr="true">font-size: 14px; color: #2d3436; font-weight: 500; margin-bottom: 5px;</string>
            </property>
            <property name="text">
             <string>Current Passphrase</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="currentPassphrase">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
  padding: 8px 12px;
  border: 1px solid #dfe6e9;
  border-radius: 6px;
  background-color: #f8f9fa;
  color: #2d3436;
  font-size: 14px;
}
QLineEdit:focus {
  border: 1px solid #6c5ce7;
  background-color: white;
}</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::EchoMode::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="newPassphraseLabel">
            <property name="styleSheet">
             <string notr="true">font-size: 14px; color: #2d3436; font-weight: 500; margin-bottom: 5px;</string>
            </property>
            <property name="text">
             <string>New Passphrase</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="newPassphrase">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
  padding: 8px 12px;
  border: 1px solid #dfe6e9;
  border-radius: 6px;
  background-color: #f8f9fa;
  color: #2d3436;
  font-size: 14px;
}
QLineEdit:focus {
  border: 1px solid #6c5ce7;
  background-color: white;
}</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::EchoMode::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="confirmPassphraseLabel">
            <property name="styleSheet">
             <string notr="true">font-size: 14px; color: #2d3436; font-weight: 500; margin-bottom: 5px;</string>
            </property>
            <property name="text">
             <string>Confirm New Passphrase</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="confirmPassphrase">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
  padding: 8px 12px;
  border: 1px solid #dfe6e9;
  border-radius: 6px;
  background-color: #f8f9fa;
  color: #2d3436;
  font-size: 14px;
}
QLineEdit:focus {
  border: 1px solid #6c5ce7;
  background-color: white;
}</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::EchoMode::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="passphraseRequirements">
            <property name="styleSheet">
             <string notr="true">font-size: 12px; color: #636e72; margin-top: 5px;</string>
            </property>
            <property name="text">
             <string>Passphrase must be between 20 and 64 characters</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="passphraseSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="passphraseButtonLayout">
            <item>
             <spacer name="passphraseHorizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="passphraseCancelButton">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>44</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
  font-size: 14px;
  font-weight: bold;
  color: #2d3436;
  background-color: #dfe6e9;
  border-radius: 10px;
  padding: 0 20px;
}
QPushButton:hover {
  background-color: #b2bec3;
}
QPushButton:pressed {
  background-color: #a4b0be;
}</string>
              </property>
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="passphraseSaveButton">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>44</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
  font-size: 14px;
  font-weight: bold;
  color: white;
  background-color: #6c5ce7;
  border-radius: 10px;
  padding: 0 20px;
}
QPushButton:hover {
  background-color: #5049c9;
}
QPushButton:pressed {
  background-color: #4040b0;
}</string>
              </property>
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="verticalLine">
         <property name="styleSheet">
          <string notr="true">background-color: #dfe6e9;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::VLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="authWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>400</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget#authWidget {
  background-color: white;
  border: 1px solid #dfe6e9;
  border-radius: 12px;
  padding: 30px;
}</string>
         </property>
         <layout class="QVBoxLayout" name="authLayout">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="authSectionTitle">
            <property name="styleSheet">
             <string notr="true">font-size: 18px; font-weight: bold; color: #2d3436; margin-bottom: 10px;</string>
            </property>
            <property name="text">
             <string>Authentication Code</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="authDescription">
            <property name="styleSheet">
             <string notr="true">font-size: 14px; color: #636e72; margin-bottom: 20px;</string>
            </property>
            <property name="text">
             <string>Enter your authentication code or scan the QR code to verify your device.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="authCodeInput">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
  padding: 8px 12px;
  border: 1px solid #dfe6e9;
  border-radius: 6px;
  background-color: #f8f9fa;
  color: #2d3436;
  font-size: 14px;
}
QLineEdit:focus {
  border: 1px solid #6c5ce7;
  background-color: white;
}</string>
            </property>
            <property name="placeholderText">
             <string>Enter authentication code</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="scanQRButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
  font-size: 14px;
  font-weight: bold;
  color: #6c5ce7;
  background-color: white;
  border: 2px solid #6c5ce7;
  border-radius: 6px;
}
QPushButton:hover {
  background-color: #f5f3ff;
}
QPushButton:pressed {
  background-color: #eeeaff;
}</string>
            </property>
            <property name="text">
             <string>Scan QR Code</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="qrCodeLabel">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #f8f9fa; border-radius: 6px;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="authSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="authButtonLayout">
            <item>
             <spacer name="authHorizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="authCancelButton">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>44</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
  font-size: 14px;
  font-weight: bold;
  color: #2d3436;
  background-color: #dfe6e9;
  border-radius: 10px;
  padding: 0 20px;
}
QPushButton:hover {
  background-color: #b2bec3;
}
QPushButton:pressed {
  background-color: #a4b0be;
}</string>
              </property>
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="authSaveButton">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>44</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
  font-size: 14px;
  font-weight: bold;
  color: white;
  background-color: #6c5ce7;
  border-radius: 10px;
  padding: 0 20px;
}
QPushButton:hover {
  background-color: #5049c9;
}
QPushButton:pressed {
  background-color: #4040b0;
}</string>
              </property>
              <property name="text">
               <string>Verify</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="bottomSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavBar</class>
   <extends>QWidget</extends>
   <header>src/ui/utils/navbar/navbar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
